app/(tabs)/(home)/index.tsx

```diff
@@ -1,39 +1,43 @@
 import React, { useState, useEffect } from "react";
 import { View, StyleSheet, Alert } from "react-native";
-import MapboxGL from "@rnmapbox/maps";
-import { Layout, Button } from "@ui-kitten/components";
+
 import * as Location from "expo-location";
+import * as TaskManager from "expo-task-manager";
+
 import {
   startAutomaticTracking,
   stopAutomaticTracking,
 } from "@/services/tracking";
-import * as TaskManager from "expo-task-manager";
+
+import MapboxGL from "@rnmapbox/maps";
+
+import { Layout, ButtonGroup } from "@ui-kitten/components";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import BigRoundButton from "@/components/Buttons/BigRoundButton";
 
 MapboxGL.setAccessToken(
   "pk.eyJ1Ijoia2F0emFibGFuY2thIiwiYSI6ImNtM2N4am40cTIyZnkydnNjODBldXR1Y20ifQ.q0I522XSqixPNIe6HwJdOg",
 );
+enum ButtonStates {
+  NotCycling,
+  Cycling,
+  Paused,
+}
 
-export default function Index() {
-  const [tracking, setTracking] = useState(false); //TaskManager.isTaskRegisteredAsync("background-location-task") PROBLEM
+export default function HomeScreen() {
+  const [, setTracking] = useState(false); //TaskManager.isTaskRegisteredAsync("background-location-task") PROBLEM
   const [latitude, setLatitude] = useState(50.0826); // Default to Wiesbaden
   const [longitude, setLongitude] = useState(8.24); // Default to Wiesbaden
+  const [buttonState, setButtonState] = useState(ButtonStates.NotCycling);
+  const buttonSize = 60;
```

Comment:
```suggestion
  const buttonIconSize = 60;
```
Vllt. das noch kurz umbenennen. Ansonsten sieht alles gut aus.